components/Table.vue

```diff
@@ -261,7 +267,19 @@
                 this.loading = true;
                 axios.get(url, { params: this.defaultParams })
                      .then(response => {
-                        this.tableData = response.data.data;
+                         if (this.showEmpty && response.data.data.length < this.paginate) {
```

Comment:
Might be safer to do a check on the response.data && response.data.data. I know this wasn't in here before, but would be useful.